.github/workflows/sdk-update-api-spec.yaml

```diff
@@ -0,0 +1,41 @@
+name: SDK Update API from from Spec
+
+on:
+  schedule:
+    - cron: '0 4 * * *'
+  workflow_dispatch:
+
+jobs:
+  api-spec-update:
+    runs-on: ubuntu-20.04
+    if: ${{ github.repository == 'jellyfin/jellyfin-sdk-kotlin' }}
+    steps:
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+      - name: Setup Gradle cache
+        uses: actions/cache@v2.1.4
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Set Stable API Version EnvVar
+        run: |
+          VERSION=$(curl -sL https://repo.jellyfin.org/releases/openapi/jellyfin-openapi-stable.json | jq -r .info.version)
+          echo "STABLE_API_VERSION=${VERSION}" >> $GITHUB_ENV
+      - name: Update generated Jellyfin API from spec
+        uses: technote-space/create-pr-action@v2
+        with:
+          EXECUTE_COMMANDS: ./gradlew --build-cache --no-daemon --info updateApiSpecStable apiDump
+          COMMIT_MESSAGE: 'update generated Jellyfin API from from spec'
```

Comment:
```suggestion
          COMMIT_MESSAGE: 'update generated Jellyfin API from spec'
```

Reply:
insert FUUUUUUU meme .... ty

Reply:
![as you wish](https://i.kym-cdn.com/entries/icons/original/000/000/063/Rage.jpg)